### PR TITLE
Update to upstream 2.10.32

### PR DESCRIPTION
**Pull request by realHenningLorenzen**

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -278,7 +278,7 @@ parts:
     after:
     - babl
     - fix-pkgconfig-files
-    source: https://download.gimp.org/pub/gegl/0.4/gegl-0.4.36.tar.xz
+    source: https://download.gimp.org/pub/gegl/0.4/gegl-0.4.38.tar.xz
     source-checksum: sha256/6fd58a0cdcc7702258adaeffb573a389228ae8f0eff47578efda2309b61b2ca6
```
Comment (diddledani) [inline snap/snapcraft.yaml:282]: You need to add the expected checksum for the new gegl tarball to this line.

**Review by diddledani — CHANGES_REQUESTED** [review]

File: snap/snapcraft.yaml (hunk 1 above)
Comment (realHenningLorenzen) [reply snap/snapcraft.yaml]: Done

**Review by diddledani — APPROVED** [review]
Looks good to me! lets see if the build check complains...

**Comment by diddledani** [comment]
I'm investigating the build failure. I think it is specific to GitHub Actions - running the same build locally is succeeding.

**Comment by diddledani** [comment]
I'm gonna go ahead and merge this anyway, as I can't get it to fail to build here... hopefully the snapcraft.io build farm will not have any issue either :-)

**Comment by linghengqian** [comment]
@diddledani Hi, 2.10.32 has been at latest/edge for a long time, can we push this version to latest/stable?  Refer to https://snapcraft.io/gimp .

**Pull request merged**